### PR TITLE
Add craiglpeters as a csi-proxy-maintainer

### PR DESCRIPTION
**Pull request by craiglpeters**

### CODE REVIEW

File: config/kubernetes-csi/org.yaml [hunk 1]
```diff
@@ -260,6 +260,7 @@ teams:
     - PatrickLang
     - saad-ali
     - xing-yang
+    - craiglpeters
```
Comment (nikhita) [inline config/kubernetes-csi/org.yaml:263]: This list needs to be alpha sorted for the tests to pass

**Comment by nikhita** [comment]
@craiglpeters friendly ping on this :)

**Comment by justaugustus** [comment]
@craiglpeters -- Ping again on this one.

**Comment by nikhita** [comment]
@craiglpeters Going to close due to lack of activity. Please reopen when it's ready again!

/close

**Pull request closed without merging**